### PR TITLE
Make binding to an interface configurable and optional + always send txs

### PR DESCRIPTION
**Pull request by Geod24**
See each commit for the motivation.

### CODE REVIEW

File: source/faucet/main.d [hunk 1]
```diff
@@ -274,8 +274,8 @@ public class Faucet : FaucetAPI
 
     void send ()
     {
-        if (!this.state.update(this.client, Height(this.state.known + 1)))
-            return;
+        if (this.state.update(this.client, Height(this.state.known + 1)))
+            logTrace("State has been updated: %s", this.state.known);
 
         if (this.state.known < 1)
             return logInfo("Waiting for setup to be completed");
```
Comment (kchulj) [inline source/faucet/main.d:282]: Should the `return` here be removed as well?
I've seen it get stuck here.

Comment (Geod24) [reply source/faucet/main.d:282]: Looking into it right now.

**Review by kchulj — APPROVED** [review]
✅

**Comment by Geod24** [comment]
Let's go with this and revisit the initial wait later.

**Pull request merged**